mappings/net/minecraft/block/TurtleEggBlock.mapping

```diff
@@ -1,3 +1,7 @@
 CLASS bnp net/minecraft/block/TurtleEggBlock
+	FIELD a ONE_EGG_SHAPE Lcnl;
+	FIELD b MULTIPLE_EGGS_SHAPE Lcnl;
```

Comment:
ONE_EGG_SHAPE/MULTIPLE_EGG_SHAPE?

Reply:
The block has a small shape when EGGS blockstate value is 1, and a bigger shape when it is 2, 3 or 4

Reply:
Why not just SMALL_SHAPE and BIG_SHAPE?

Reply:
To be more accurate, but could be good too